rules/explicitOverrideRule.ts

```diff
@@ -37,6 +39,8 @@ interface IOptions {
     useJsdocTag: boolean;
     useDecorator: boolean;
     excludeInterfaces: boolean;
+    lowercase: boolean;
+    uppercase: boolean;
```

Comment:
Change that to `decoratorCase: 'pascal' | 'camel' | 'any'` to better represent the possible states (Two booleans is 4 states, but there are only 3 possible states here)

Reply:
for reference: [PascalCase , camelCase , snake-case](https://en.wikipedia.org/wiki/Camel_case)

Reply:
Are we sure we want to have `'any'` there? I have it defaulting to `@override` if `@Override` isn't provided right now, so technically there are only the two states internally. `IOptions.uppercase` isn't even used. Once the other changes are reverted, it will make it so that all this code does is change the fixer, so `any` makes even *less* sense. How do you insert `@override` in "any" case?

So technically it could be like `usePascalCaseDecorator: boolean` (or `useUppercaseDecorator`)

Reply:
Yes that is correct. See my other reply.